src/api/event/content-types/event/schema.json

```diff
@@ -34,6 +32,19 @@
       "type": "relation",
       "relation": "oneToOne",
       "target": "plugin::users-permissions.user"
+    },
+    "action": {
+      "type": "enumeration",
+      "enum": [
+        "Idea Created",
+        "Idea Updated",
+        "Commented",
+        "Liked",
+        "Test"
```

Comment:
We are leaving this in for the next month to give us the ability to test, and will remove it in the future.